Programs/YankeeFork/Program.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../Program.xsd">
+
+  <Name>Yankee Fork</Name>
+  <MachineName>YankeeFork</MachineName>
+
+  <MetaData>
+    <Meta name="projectFile">project.rs.xml</Meta>
+    <Meta name="access">PRIVATE</Meta>
+    <Meta name="apiUrl">https://iz2u8e5df2.execute-api.us-west-2.amazonaws.com/prod/api</Meta>
+    <Meta name="projectXSDs">http://xml.riverscapes.xyz/Projects/XSD/V1</Meta>
+  </MetaData>
+
+  <Definitions>
+    <!--Here's where we say what's allowed for each collection. Can be a list or a regex-->
+    <Collections>
+      <Collection id="COL_WATERSHED" name="Watershed">
+        <Allow id="YankeeFork" name="YankeeFork" folder="YankeeFork" />
+        <Allow id="Yankee Fork" name="Yankee Fork" folder="YankeeFork" />
```

Comment:
I wouldn't double up here. Choose one and change your project Meta tags to use it. Otherwise things get weird.